core/templates/donors.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block title %}Apoiadores - Brasil.IO{% endblock %}
+{% block head %}
+{{ block.super }}
+<!-- Infinte scroll -->
+<script src="{% static 'js/noframework.waypoints.min.js' %}"></script>
+<script src="{% static 'js/infinite.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div class="section">
+  <h4>Doadores</h4>
+  <div class="divider"></div>
+
+  <div class="row m-t-5">
+    <p>
+      A lista abaixo é de pessoas que <i>apoiam nossa campanha</i> pela plataforma <strong>Apoia.se</strong> de
+      financiamento coletivo.
+      <a href="https://apoia.se/brasilio" title="Apoie nossa campanha">Colabore você também clicando aqui.</a>
+    </p>
+    <div class="row">
+      <div class="infinite-container col s12">
+        {% for donor in donors %}
+          <div class="col s12 m3 m-t-15 infinite-item center" style="min-height: 170px;">
+            <span class="fa-stack fa-3x text-center">
+              <img class="circle responsive-img" src="{{ donor.image }}" alt="Imagem de {{ donor.name }}">
```

Comment:
Esse é um "erro" muito específico para minha pessoa. Como eu uso a extensão [Facebook Container](https://addons.mozilla.org/en-US/firefox/addon/facebook-container/) do Firefox, todo o tráfego para o Facebook é bloqueado se eu não estiver dentro do container do Facebook. Faço isso para evitar tracking por parte deles. 

O efeito colateral aqui nessa página é o de que imagens provenients de graph.facebook, como [esta (https://graph.facebook.com/v3.0/1717683618353950/picture), não são apresentadas. Isso em si não é o problema, mas a maneira com que o `alt` está sendo apresentado fica estranha como dá para ver aqui:

![Screenshot from 2020-09-09 11-32-14](https://user-images.githubusercontent.com/238223/92614368-59905680-f292-11ea-98a3-e61ccd4a77c4.png)

Reply:
Acho que aqui podemos fazer duas coisas:
- Corrigir o alt da imagem
- Usarmos a app de thumbnail (sorl-thumbnail) para essas imagens, assim elas ficarão no tamanho que queremos e, de quebra, como o Brasil.IO vai servi-las, quem bloqueia Facebook não terá esse problema.

Reply:
Boa @turicas! O sorl-thumbanail vai funcionar muito bem aqui!

@marcmatias só para você ter como referência, no [HTML dos boletins tem um exemplo](https://github.com/turicas/brasil.io/blob/develop/covid19/templates/covid19/bulletins.html#L39) do que o Turicas tá falando. Pra ele funcionar, é necessário carregar a tag colocando o [`{% load thumbnail %}`](https://github.com/turicas/brasil.io/blob/develop/covid19/templates/covid19/bulletins.html#L5) no início do arquivo.

Reply:
> @marcmatias só para você ter como referência, no [HTML dos boletins tem um exemplo](https://github.com/turicas/brasil.io/blob/develop/covid19/templates/covid19/bulletins.html#L39) do que o Turicas tá falando. Pra ele funcionar, é necessário carregar a tag colocando o [`{% load thumbnail %}`](https://github.com/turicas/brasil.io/blob/develop/covid19/templates/covid19/bulletins.html#L5) no início do arquivo.

Foi o próprio @marcmatias que [introduziu o sorl-thumbnail no Brasil.IO](https://github.com/turicas/brasil.io/pull/349) 😂

Reply:
Eita!! Faltou eu dar um `git blame` então heheheh

Reply:
Modificado, puxando imagens com sorl-thumbanail e corrigido o erro que ocorria no Firefox